cellprofiler/modules/morphologicalskeleton.py

```diff
@@ -28,6 +30,9 @@ class MorphologicalSkeleton(cellprofiler.module.ImageProcessing):
 
     variable_revision_number = 1
 
+    def volumetric(self):
```

Comment:
Wow, awesome work porting this module to 3D! :stuck_out_tongue_winking_eye:

Reply:
🙃